docs/faq.md

```diff
@@ -0,0 +1,16 @@
+# <img align="center" src="./images/logo.png">  FAQ
+
+1. What version of AutoRest C# should I use?
```

Comment:
Should we also add this question?

>How do I generate code based on V2 version?
   You can add --legacy flag to the command line to get the previous behavior.